pkg/test/istio.io/examples/scriptTestType.go

```diff
@@ -0,0 +1,141 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package examples
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+	"testing"
+
+	testEnv "istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/scopes"
+)
+
+type scriptTestType struct {
+	script   string
+	output   outputType
+	verifier VerificationFunction
+}
+
+func newStepScript(script string, output outputType, verifier VerificationFunction) testStep {
+	return scriptTestType{
+		script:   script,
+		output:   output,
+		verifier: verifier,
+	}
+}
+
+func (test scriptTestType) Run(env *kube.Environment, t *testing.T) (string, error) {
+	scopes.CI.Infof(fmt.Sprintf("Executing %s\n", test.script))
```

Comment:
No need to use fmt.Spritntf here, just call Infof directly.

Reply:
Fixed. Thanks.